jenkinsbuilder/weekly-scan.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+import beanstalkc
+import json
+import os
+import subprocess
+import sys
+import yaml
+
+
+# connect to beanstalkd tube
+bs = beanstalkc.Connection("BEANSTALK_SERVER")
+bs.use("master_tube")
+
+# registry server value to be replaced by ansible
+registry = "JENKINS_SLAVE"
+
+# cli argument pointing to directory containing yml files
+index_dir = sys.argv[1]
+
+# string representation of the catalog on r.c.o
+str_catalog = subprocess.check_output(
+    ["curl", "%s:5000/v2/_catalog" % registry]
+)
+
+# convert string into a list to use for comparison
+json_catalog = json.loads(str_catalog).values()[0]
+
+# have a list of files in the index_dir
+for directory, dirnames, filenames in os.walk(index_dir):
+    files = filenames
+
+# index dir will always have yml files only; but just in case
+for f in files:
+    if f.endswith(".yml"):
+        continue
+    files.remove(f)
+
+# parse the yml file
+for f in files:
+    with open(os.path.join(os.environ.get("CWD"), "index.d", f)) as stream:
+        try:
+            yaml_parse = yaml.load(stream)
+        except yaml.YAMLError as e:
+            print e
```

Comment:
I think we should log it to some file when the NFS is up. Because this doesn't run as a service. It runs as a one time script once in a week. WDYT?

With the `print` statement, we'll have record in the Jenkins console output for a build.